shared/bytes/bytes.go

```diff
@@ -45,3 +45,15 @@ func Bytes8(x uint64) []byte {
 func FromBytes8(x []byte) uint64 {
 	return binary.BigEndian.Uint64(x)
 }
+
+// LowerThan returns true byte slice x is lower than byte slice y.
+// This is used in spec to compare winning block root hash.
+// Mentioned in spec as "ties broken by favoring lower `shard_block_root` values".
+func LowerThan(x []byte, y []byte) bool {
+	for i, b := range x {
```

Comment:
Is this the correct way to compare whichever byte slice is lower ? 
What if  `x[0] < y[0], x[1] > y[1] , x[2] < y[2]` , wouldnt x still be lower ? I think
we are supposed to convert the blockhash to a big.Int and then compare them

Reply:
Assume 0 is the most significant bit, x will always be lower and we don't need to check for 1 and 2. I can't think of why it wouldn't be lower. 

Here is the reference implementation from Nim:
https://github.com/status-im/nim-beacon-chain/blob/master/beacon_chain/state_transition.nim#L405

Reply:
In that case , can you make add a comment that we are assuming the big-endian format here.

Reply:
sure thing!